doc/filter_tutorial.md

```diff
@@ -0,0 +1,376 @@
+# Alerts tutorial
+
+This document will help you navigate through the simulated Rubin alerts ingested in the MongoDB Compass, so that you can try different filtering schemes using the MongoDB syntax, which is compatible with Fritz.
+First a summary of what you’ll do:
+- Install MongoDB Compass
+- Connect to cluster with Rubin alerts
+- Create your filter
+- Save your candidates
+- Explore your results
+
+### 0. Note
+I based most of this work on the tutorial you can find here, so, if you have any issues, I would start looking them up there.
+### 1. To install MongoDB Compass
+ Go to https://www.mongodb.com/products/compass and click in ‘Download now’
+### 2. Connecting to cluster
+Once Mongo DB Compass is installed, launch it, and connect to the cluster by
+
+   - Opening Mongo DB Compass
+   - Copy the following into the box under New Connection, and click Connect.  <br/>
+mongodb+srv://alert_digger:WhereIsTheKN@cluster0.s0ada.mongodb.net/test  <br/>
+
+
+     ![](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial1.png)
+
+
+
+ - To see the alerts (around 4000 of them) navigate in the left menu to the folder ZTF > lsst_800
+
+   ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial2.png)
+
+
+
+ **Tab description**
+
+- Documents: here you can see all the avro packages, and their individual properties. You can add Data or export a collection. You can apply filters here too, but it is easier in the Aggregation tab. It displays only 20 alerts.
+- Aggregations: Here you can make a new filter and apply it to the collection. After each step, you can visualize how many alerts passed the filters and modify the filter step to check how things change. You can export the results and resulting filter.
+- Scheme: tbd
+- Explain plan: it shows the details of your filter
+- Indexes: tbd
+- Validation: tbd
+
+### 3. To make a new filter
+In the Aggregation tab, click on Create New > Pipeline from text. This will promt a new window where you can write/paste your filter.
+
+ ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial3.png)
```

Comment:
Should we use a better title than "This is an image"?

Reply:
changed the titles